skema/program_analysis/fn_unifier.py

```diff
@@ -221,6 +221,14 @@ def align_full_system(gromet_obj, extraction, extraction_file_name):
                         for FN in module_FN.fn_array:
                             align_fn(FN_metadata, file_comments, FN)
         
+def process_alignment(gromet_json, comments_json):
```

Comment:
@titomeister , can you please add some typehints to this?

Reply:
i.e., (`gromet_json: Dict[Text, Any]`, `comments_json: Dict[Text, Any]` -> GrometFNModuleCollection:`).  Idk if we can be any more specific in the typehint about the structure of `comments_json`.

Reply:
Done!